scripts/integration_test_translator.py

```diff
@@ -240,7 +242,8 @@ def main() -> None:
     success = True
     for t in tests:
         logging.debug("running test: %s", t.__name__)
-        success = success and t(args)
+        if not t(args):
+            success = False
```

Comment:
Why doesn't the old version type-check?

Reply:
Because `test_minimal` and `test_hello_world` returned None. They should really return bools propagated from their inner calls, though--I've changed them to do so, which obviates this change.